alphagov/paas-product-pages
# Setup Jest test runner + button component tests

## What

Setup a testing framework:

- Setup app to use Jest and [Testing Library](https://testing-library.com/) to run tests.
- Remove redundant button component functionality we don't use
- Add button component tests
- Add Github Actions to run test on Pull Request

## How to test

- clone branch
- run `npm install` to install latest dependencies
-  run `npm test` to verify
- review code
